src/flight_booking_agent/graph/workflow.py

```
@@ -15,7 +15,9 @@
 
 from ..tools.flight_tools import search_flights_tool
 from ..tools.booking_tools import create_booking_tool 
-# (Giả định bạn sẽ tạo file booking_tools.py)
+from ..agents.flight_searcher import flight_search_node # Import node mới
```

Ruff (0.13.2)

Redefinition of unused `flight_search_node` from line 11

Remove definition: `flight_search_node`

(F811)